misc/ballerina-formatter/src/main/java/org/ballerinalang/format/FormattingCmd.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.format;
+
+import org.ballerinalang.launcher.BLauncherCmd;
+import picocli.CommandLine;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * Class to implement "format" command for ballerina.
+ * Ex: ballerina format [ballerinaFile|ModuleName] [-ow|--over-write]
+ */
+@CommandLine.Command(name = "format", description = "format given Ballerina source file")
+public class FormattingCmd implements BLauncherCmd {
```

Comment:
Should this be `FormatCmd` instead to be inline with our other commands `BuildCmd`, `RunCmd`, etc.?

Reply:
yes that makes sense... will do the change